tests/run-custom-args/captures/colltest5/Test_2.scala

```diff
@@ -76,7 +76,7 @@ object Test {
     val x4 = xs.head
     val y4: Int = x4
     val x5 = xs.to(List)
-    val y5: {x5} List[Int] = x5
+    val y5: List[Int] = x5
```

Comment:
It's because `List` is a pure class, which means no instance of `List` could capture. See https://github.com/lampepfl/dotty/pull/16254/commits/adbf28051bb3292a436b8f902e1e6bb2f202f149.

Reply:
Ah I get that. Thanks for your explanation! :)